.github/workflows/gr-python-release.yml

```diff
@@ -0,0 +1,136 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: GR Python
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'maint-*'
+      - '[0-9]+.[0-9]+.[0-9]+-gr'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+-gr.[0-9]+'
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  python:
+    name: Python ${{ matrix.python_version }}
+    runs-on: ubuntu-latest
```

Comment:
We probably want to build with an older version of Ubuntu to ensure the built libraries will be compatible with older versions of glibc, otherwise they'll only work on Ubuntu 24.04 or newer. I'm not sure what the oldest glibc is we'd want to use internally though, maybe ubuntu-22.04 should be fine.

The official wheels are built on a [manylinux](https://github.com/pypa/manylinux) CentOS Docker image. I think we should be fine to build on Ubuntu though.

Reply:
Will move to Ubuntu 22.04 then. Ubuntu 20.04 will be removed from Github in the next few weeks.